static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -0,0 +1,323 @@
+import {Component} from 'react';
+import {createFilter} from 'react-select';
+import debounce from 'lodash/debounce';
+
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {Client} from 'app/api';
+import {t} from 'app/locale';
+import {SentryAppInstallation} from 'app/types';
+import {replaceAtArrayIndex} from 'app/utils/replaceAtArrayIndex';
+import withApi from 'app/utils/withApi';
+import FieldFromConfig from 'app/views/settings/components/forms/fieldFromConfig';
+import Form from 'app/views/settings/components/forms/form';
+import FormModel from 'app/views/settings/components/forms/model';
+import {Field, FieldValue} from 'app/views/settings/components/forms/type';
+
+// 0 is a valid choice but empty string, undefined, and null are not
+const hasValue = value => !!value || value === 0;
+
+export type FieldFromSchema = Omit<Field, 'choices' | 'type'> & {
+  type: 'select' | 'textarea' | 'text';
+  default?: 'issue.title' | 'issue.description';
```

Comment:
FieldFromSchema.default was changed to match the only expected values from [the docs](https://docs.sentry.io/product/integrations/integration-platform/ui-components/#attributes-1)